starter-code/src/clue.js

```diff
@@ -85,10 +85,124 @@ name: Patio
 */
 
 // Characters Collection
-var charactersArray = [];
+var charactersArray = [
+    {
+    mrGreen: {
```

Comment:
proper syntax for objects is: 
```
var charactersArray = [
  var mrGreen = { ... }
]
```
and so on.. When you change this, you'll have working code.